tests/bson/bug0531-002.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+PHPC-531: Segfault due to double free by corrupt BSON visitor (nested document)
```

Comment:
It's a bit odd for this to add a new PHPC-531 test; however, `bson-toPHP_error-004.phpt` is the only other place where we test for "Detected unknown BSON type" outside of BSON spec tests. 

Ideally, this would be a new test case in that file (using `pack()`), or a newly created `bson-toPHP_error-005.phpt` file (there is a gap between 4 and 6). The latter would be fairly low effort, but I won't argue if you want to leave this as-is.